ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/util/SampleData.java

```diff
@@ -74,6 +75,50 @@
 
 public class SampleData {
 
+    private static Supplier<CCDCase.CCDCaseBuilder> getCCDBuilderWithDefault = () -> CCDCase.builder()
```

Comment:
minor suggestion: maybe ename to 'ccdBuilderWithDefault' because supplier has a get() method by default

Reply:
have made changes